tests/cypress/support/commands.ts

```diff
@@ -1004,3 +1004,39 @@ Cypress.Commands.add('moveClusterToWorkspace', (clusterName, workspaceName, time
 Cypress.Commands.add('restoreClusterToDefaultWorkspace', (clusterName, timeout, defaultWorkspaceName='fleet-default', ) => {
   cy.moveClusterToWorkspace(clusterName, defaultWorkspaceName, timeout);
 })
+
+Cypress.Commands.add('k8sUpgrade', (clusterName) => {
```

Comment:
here, given that this may affect also the local cluster, perhaps to avoid confusion with other upgrades could be renamed as `k8sUpgradeInRancher`

Reply:
I will  update it.